### PR TITLE
Address feature feedbacks

### PR DESCRIPTION
**Pull request by annakocheshkova**
Address the feedbacks on the main PR:
- Rewrite McFusUploader to class
- Remove extra package
- Move helper functions

[AB#80767](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/80767)

### CODE REVIEW

File: src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts [hunk 1]
```diff
@@ -16,7 +16,8 @@ import {
 import * as fs from "fs";
 import fetch from "node-fetch";
 import { MimeTypes } from "./mc-fus-mime-types";
-const Path = require("path");
+import * as Path from "path";
+import "abort-controller/polyfill";
```
Comment (MatkovIvan) [inline src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:20]: Use regular import, not polyfill

File: src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts [hunk 2]
```diff
@@ -53,21 +54,25 @@ class HttpError extends Error {
     this.statusText = statusText;
   }
 }
+export interface McFusUploader {
```
Comment (MatkovIvan) [inline src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:57]:
```suggestion

export interface McFusUploader {
```

Comment (MatkovIvan) [reply src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:57]: Should we move it to -types file?

File: src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts [hunk 3]
```diff
@@ -53,21 +54,25 @@ class HttpError extends Error {
...
+  uploadBaseUrls: any = {
     CancelUpload: "upload/cancel/",
     SetMetadata: "upload/set_metadata/",
     UploadChunk: "upload/upload_chunk/",
     UploadFinished: "upload/finished/",
     UploadStatus: "upload/status/",
   };
 
-  const uploadData: IUploadData = {
+  uploadData: IUploadData = {
```
Comment (MatkovIvan) [inline src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:75]: Why do you remove const here?

File: test/commands/distribute/release/release-test.ts [hunk 4]
```diff
@@ -846,14 +846,6 @@ describe("release command", () => {
       });
   }
 
-  function setupFailedCreateReleaseResponse(nockScope: Nock.Scope): Nock.Scope {
```
Comment (MatkovIvan) [inline test/commands/distribute/release/release-test.ts:849]: Why is it removed?

Comment (annakocheshkova) [reply test/commands/distribute/release/release-test.ts:849]: Unused

Comment (annakocheshkova) [reply test/commands/distribute/release/release-test.ts:849]: Must have been left from previous implementation

File: src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts (hunk 3 above)
Comment (annakocheshkova) [reply src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:75]: There are no "const" class members in typescript. I can only make it `readonly`.

File: src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts (hunk 1 above)
Comment (annakocheshkova) [reply src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts:21]: Somehow it doesn't work that way. I have no context on this fix so dropping it as is for now to prioritize other work.

**Pull request merged**